### PR TITLE
refactor(fivetran_sdk): improvements in examples

### PR DESCRIPTION
**Pull request by fivetran-satvikpatil**
Closes https://fivetran.height.app/T-718405

### Description of change
- Added a field missing in VisibilityCondition
- Added example usage of FieldSet

### CODE REVIEW

File: common.proto [hunk 1]
```diff
@@ -111,6 +112,13 @@ enum DataType {
   JSON = 15;
 }
 
+message DataTypeParams {
```
Comment (ediril) [inline common.proto:115]: This is not in production yet, we shouldn't add it to `common.proto`. It should go into `common_v2.proto`

File: common.proto [hunk 2]
```diff
@@ -153,5 +161,5 @@ message Column {
   string name = 1;
   DataType type = 2;
   bool primary_key = 3;
-  optional DecimalParams decimal = 4;
+  optional DataTypeParams params = 4;
```
Comment (ediril) [inline common.proto:164]: Not in production yet

File: destination_sdk.proto [hunk 3]
```diff
@@ -24,6 +24,7 @@ message CapabilitiesRequest {}
 message CapabilitiesResponse {
     repeated DataTypeMappingEntry data_type_mappings = 1;
```
Comment (ediril) [inline destination_sdk.proto:25]: In production, we only have `BatchFileFormat batch_file_format = 1;`

File: destination_sdk.proto [hunk 4]
```diff
@@ -24,6 +24,7 @@ message CapabilitiesRequest {}
...
     bool supports_history_mode = 2;
+    BatchFileFormat batch_file_format = 3;
 }
 
 message DataTypeMappingEntry {
```
Comment (ediril) [inline destination_sdk.proto:30]: This is not in production yet

**Review by ediril — CHANGES_REQUESTED** [review]
This PR should be modifying/using `xxx_v2.proto` files and the examples should also be using the v2 proto files. Instead it is modifying the production proto files. Can we please fix it?

**Comment by fivetran-satvikpatil** [comment]
@ediril I have made these changes in this PR: https://github.com/fivetran/fivetran_sdk/pull/50. 
As it was dependent on Manju's PR. I am closing this PR.

**Pull request closed without merging**